libs/api-sdk/src/index.test.ts

```diff
@@ -67,14 +66,14 @@ describe("Bandada API SDK", () => {
                 baseURL: SupportedUrl.PROD
             }
             const fun = () => {
-                apiSdk = new ApiSdk(SupportedUrl.DEV, config)
+                const apiSdk: ApiSdk = new ApiSdk(SupportedUrl.DEV, config)
             }
             expect(fun).toThrow("The url and baseURL should be the same")
         })
         it("Should throw an error when the url has the wrong type", () => {
             const url = 123
             const fun = () => {
-                apiSdk = new ApiSdk(url as any)
+                const apiSdk: ApiSdk = new ApiSdk(url as any)
```

Comment:
To fix the lint error here, it would be the same as the previous idea, update the `fun` function with this:

```ts
const fun = () => new ApiSdk(url as any)
```

Reply:
Done